src/Pages/MapPage/Map.tsx

```
@@ -0,0 +1,471 @@
+import { fetchMissionReviews, type Review } from "../../api/reviews";
+import { fetchMissions, type Mission } from "../../api/mission";
+import duck from "../../assets/duck.png";
+import { ReviewHero } from "./Review";
+import { api } from "../../api/client";
+import { useNavigate } from "react-router-dom";
+import alert from "../../assets/alert.png";
+
+type SliderLevel = "closed" | "half" | "full";
+type Tab = "mission" | "review";
+
+const Map: React.FC = () => {
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)